BlockID fixed

fernanACM opened the pull request:
* BlockID identifier, will now detect aliased IDs
* Fix in "Removeblock" subcommand
* Version: 0.0.3

fernanACM committed:
BlockID fixed

KingOfTurkey38 commented:
Has this been properly tested?

fernanACM commented:
> Has this been properly tested?

Yes, it works correctly

Endermanbugzjfc reviewed (changes requested):
Doesn't it only detect the first alias?

fernanACM committed:
Delete .poggit.yml

poggit-bot committed:
Create .poggit.yml
Poggit-CI is enabled for this repo by @fernanACM
Visit the Poggit-CI page for this repo at https://poggit.pmmp.io/ci/ACM-PocketMine-MP/MineReset38

fernanACM committed:
Fixed

fernanACM committed:
Update .poggit.yml

fernanACM committed:
Update plugin.yml
